main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -28,7 +28,7 @@ object Verifier {
 
   def run(root: Root)(implicit flix: Flix): Root = flix.phase("Verifier") {
     if (!flix.options.xnoverify) {
-      root.defs.values.foreach(d => visitDef(d)(root))
+      ParOps.parMap(root.defs.values)(visitDef(_)(root))
```

Comment:
Is there not a better parMap thing or  do we have to construct the .values thing?

Reply:
Its not super clear from the docs, but values is `Iterable` and is based on `valuesIterator` so I'm pretty sure its not materialized.

There is no wellsuited ParOps things - it doesn't have foreach style methods

Reply:
Ok